apps/settings/elements/wifi.html

```diff
@@ -45,10 +44,9 @@ <h2 data-l10n-id="advancedSettings"></h2>
       </header>
       <ul>
         <li class="hint">
-          <label class="pack-switch">
-            <input type="checkbox" name="wifi.sleepMode">
-            <span data-l10n-id="wifi-sleep"></span>
-          </label>
+          <gaia-switch name="wifi.sleepMode">
+            <label data-l10n-id="wifi-sleep"></label>
+          </gaia-switch>
```

Comment:
click the description part wont toggle the switch, seem like related to bug 1190548